bin/rubocop-quick

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+git diff --name-status --staged | grep '^[MA]' | grep -o '\s\+.*rb' | xargs bundle exec rubocop --except Metrics --auto-correct --format quiet && \
```

Comment:
I noticed the v3-website version of this script had a `--force-exclusion app/controllers/application_controller.rb` argument. Was that the argument required for the speedup of this command? In other words: should I also add an exclusion?

Reply:
Yes. Probably `Gemfile`. Its so the command doesn't fail if xargs is empty because the git doesn't have any added or removed Ruby files.